make/buf/all.mk

```diff
@@ -150,6 +152,13 @@ bufrelease:
 .PHONY: gofuzz
 gofuzz: $(GO_FUZZ)
 	@rm -rf $(TMP)/gofuzz
-	@mkdir -p $(TMP)/gofuzz
-	cd internal/buf/bufbuild/bufbuildtesting; go-fuzz-build -o $(abspath $(TMP))/gofuzz/gofuzz.zip
+	@mkdir -p $(TMP)/gofuzz $(TMP)/gofuzz/corpus $(TMP)/gofuzz/crashers
+	# go-fuzz-build requires github.com/dvyukov/go-fuzz be in go.mod, but we don't need that dependency otherwise.
+	# This adds go-fuzz-dep to go.mod, runs go-fuzz-build, then restores go.mod.
+	cp go.mod go.mod.bak; cp go.sum go.sum.bak
```

Comment:
Without the trap, if the next line fails, we now have leftover files that are not .gitignored. You can copy them to `$(TMP)` to make sure they're ignored, or re-add the script.